src/ceph-daemon/ceph-daemon

```diff
@@ -1574,6 +1584,10 @@ def _get_parser():
         '--cluster',
         default='ceph',
         help='cluster name')
+    parser_adopt.add_argument(
+        '--base-dir',
```

Comment:
``--legacy-base-dir``?

Reply:
Or maybe just shorten to `--legacy-dir` ?